scripts/utils/forkRopsten.js

```diff
@@ -0,0 +1,46 @@
+const Ganache = require("ganache-core");
```

Comment:
Lets add a CLI param to pass in the network flag for the existing `forkMainnet.js`